lib/runners/processor/pylint.rb

```diff
@@ -0,0 +1,86 @@
+module Runners
+  class Processor::Pylint < Processor
```

Comment:
[suggestion] How about including `Python` module? See also `processor/flake8.rb`

```suggestion
  class Processor::Pylint < Processor
    include Python

```